wweval/R/supplement_analyses_and_figs.R

```diff
@@ -665,15 +665,63 @@ get_stats_improved_forecasts <- function(scores,
       "location", "forecast_date"
     )) |>
     dplyr::mutate(
-      pct_change_crps = (ww - hosp) / hosp
+      pct_change_crps = (ww - hosp) / hosp,
+      rel_crps = ww / hosp
+    )
+
+  relative_crps_raw <- scores |>
+    compute_relative_crps(id_cols = c(
+      "location", "forecast_date", "date"
+    )) |>
+    dplyr::mutate(
+      pct_change_crps = (ww - hosp) / hosp,
+      rel_crps = ww / hosp
```

Comment:
Why is there a call to a `compute_relative_crps` functino followed by an explicit computation

Reply:
probably a mistake -- the way that `compute_relative_crps` works is no longer how we are presenting the results so I can look into this and make sure we are summing over the scores in the state and then computing, rather than computing for each day and averaging.

Reply:
Yeah this is redundant